app/application/route.js

```diff
@@ -38,7 +52,7 @@ export default class ApplicationRoute extends Route {
       machine: 'machine 1',
       title: 'Problem with air cylinder',
       downtime: '30',
-      createdDate: '10.05.2022',
+      createdDate: '05.10.2022',
```

Comment:
Better way is keep date data like this: `new Date('05/10/2022')`. For proper display this date in templates use helpers - you can try write by yourself

Also consider to change name this property to `createdAt` - all database support that name, it will be better for future migrate from localstorage to some external service